### ISSUE TITLE
METEOR@1.3-rc.8 Windows adds an additional C\ to the path where node_modules not found

### ISSUE THREAD

**Issue by lucidprogrammer** [meteor/meteor#6568]
METEOR@1.3-rc.8 fails on Windows 10 64 . Error is as follows. Looks like instead of C:\Users\, it looks for C:\C\Users\

```
W20160323-03:46:17.605(0)? (STDERR)
W20160323-03:46:17.606(0)? (STDERR) fs.js:696
W20160323-03:46:17.606(0)? (STDERR)   return binding.stat(pathModule._makeLong(path));
W20160323-03:46:17.606(0)? (STDERR)                  ^
W20160323-03:46:17.606(0)? (STDERR) Error: ENOENT, no such file or directory 'C:\C\Users\projects\my\src\webapp\node_modules'
W20160323-03:46:17.606(0)? (STDERR)     at Object.fs.statSync (fs.js:696:18)
W20160323-03:46:17.606(0)? (STDERR)     at findAppDir (C:\Users\projects\my\src\webapp\.meteor\local\build\programs\server\npm-require.js:9:10)
W20160323-03:46:17.607(0)? (STDERR)     at addByPath (C:\Users\projects\my\src\webapp\.meteor\local\build\programs\server\npm-require.js:58:22)
W20160323-03:46:17.607(0)? (STDERR)     at registerNodeModules (C:\Users\projects\my\src\webapp\.meteor\local\build\programs\server\npm-require.js:45:5)
W20160323-03:46:17.607(0)? (STDERR)     at C:\Users\projects\my\src\webapp\.meteor\local\build\programs\server\npm-require.js:37:9
W20160323-03:46:17.607(0)? (STDERR)     at Array.forEach (native)
W20160323-03:46:17.607(0)? (STDERR)     at Function._.each._.forEach (C:\Users\Lucid\AppData\Local\.meteor\packages\meteor-tool\1.2.5-rc.8\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\underscore\underscore.js:79:11)
W20160323-03:46:17.607(0)? (STDERR)     at Object.<anonymous> (C:\Users\projects\my\src\webapp\.meteor\local\build\programs\server\npm-require.js:30:3)
W20160323-03:46:17.607(0)? (STDERR)     at Module._compile (module.js:456:26)
W20160323-03:46:17.607(0)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
=> Exited with code: 8
```

**Comment by rynti** [comment]
Experiencing the same issue here, temporarily circumvented the error by making a folder called C on the C drive, linking to the C drive :laughing:

**Comment by AnnotatedJS** [comment]
And the same here, however adding a D, as my repository is on the D drive.

fs.js:696
  return binding.stat(pathModule._makeLong(path));
                 ^
Error: ENOENT, no such file or directory 'D:\D\github\meteor-react-start-app\node_modules'
    at Object.fs.statSync (fs.js:696:18)
    
[rest of error message is clipped]

**Comment by tomRedox** [comment]
Same issue for me too.

**Comment by benjamn** [comment]
Reproduced, thanks for the reports! Working on it now.

**Comment by tomRedox** [comment]
This is fixed for me in rc.9

**Comment by lucidprogrammer** [comment]
this is fixed. Plus the build time has improved. excellent @benjamn

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.10`](https://github.com/meteor/meteor/issues/6266#issuecomment-200906878).
